pkg/operator/operator.go

```diff
@@ -223,11 +228,27 @@ func (o *Operator) WithControllers(ctx context.Context, controllers ...controlle
 	return o
 }
 
-func (o *Operator) WithWebhooks(ctx context.Context, ctors ...knativeinjection.ControllerConstructor) *Operator {
+func (o *Operator) WithWebhooks(ctx context.Context, ctors ...knativeinjection.NamedControllerConstructor) *Operator {
 	if !options.FromContext(ctx).DisableWebhook {
 		o.webhooks = append(o.webhooks, ctors...)
 		lo.Must0(o.Manager.AddReadyzCheck("webhooks", webhooks.HealthProbe(ctx)))
 		lo.Must0(o.Manager.AddHealthzCheck("webhooks", webhooks.HealthProbe(ctx)))
+
+		kCtx, _ := knativeinjection.EnableInjectionOrDie(ctx, o.GetConfig())
+		kCtx = injection.WithClient(kCtx, o.GetClient())
+		kCtx = injection.WithNodeClasses(kCtx, []schema.GroupVersionKind{})
```

Comment:
Why do we add in the nodeclasses key in here with an empty kind? Shouldn't this be done in the cloud provider provider?

Reply:
I'm confused though, shouldn't we add in the cp.GetSupportedNodeClasses() then?

Reply:
I mean -- we could, bit it's not needed. What's needed is the key to be there -- what is stored in the key doesn't matter here for this use-case